sandpack-react/src/styles/index.css

```diff
@@ -137,8 +137,8 @@
   padding-right: var(--sp-space-2);
 }
 
-.sp-cm.sp-pristine .cm-scroller {
-  overflow: hidden;
```

Comment:
@alexnm do you recall the reason why this `overflow`? Actually, this is causing some scroll issues on the desktop as well, turns out it is very hard to focus on the div and scroll it. Once I removed it, both desktop and mobile were working as expected, but I can't see what is side-effects of this change.

Reply:
Yes, I remember. this was added to keep the editor from hijacking the scroll as you scroll into the page.

Reply:
ok, then it's safe to remove this